UnitTests/meson.build

```diff
@@ -29,23 +31,31 @@ su2_cfd_tests_dd = files(['Common/simple_directdiff_test.cpp'])
 
 if get_option('enable-tests')
   if get_option('enable-normal')
+    unit_test_deps = [
+        fs.copyfile('SU2_CFD/fluid/multicomponent_cantera.cfg', 'multicomponent_cantera.cfg'),
```

Comment:
Hi Guus, it is Cristopher from TU Eindhoven, Thanks for this pull request. Yesterday, I mentioned that Cantera is still not available in SU2, so I think the multicomponent_cantera.cfg file should be removed for now. When the Cantera implementation is ready, I will add it. Thanks for your help !!!

Reply:
Good point; that shouldn't be here for this branch. Thanks!